src/snowcli/cli/snowpark/package/manager.py

```diff
@@ -0,0 +1,101 @@
+
+        if package_response.snowflake and not package_response.other:
+            return InAnaconda(package_response)
+        else:
+            if install_packages:
+                status, result = utils.install_packages(
+                    perform_anaconda_check=True, package_name=name, file_name=None
+                )
+
+                if status:
+                    if result.snowflake:
+                        return RequiresPackages(result)
+                    else:
+                        return NotInAnaconda(result)
+
+        return NothingFound(SplitRequirements([], []))
+
+    def upload(self, file: Path, stage: str, overwrite: bool):
+        conn = snow_cli_global_context_manager.get_connection()
+
+        log.info(f"Uploading {file} to Snowflake @{stage}/{file}...")
+        with tempfile.TemporaryDirectory() as temp_dir:
+            temp_app_zip_path = utils.prepare_app_zip(file, temp_dir)
+            deploy_response = conn.upload_file_to_stage(
+                file_path=temp_app_zip_path,
+                destination_stage=stage,
+                path="/",
+                database=conn.ctx.database,
+                schema=conn.ctx.schema,
+                overwrite=overwrite,
+                role=conn.ctx.role,
+                warehouse=conn.ctx.warehouse,
+            )
+
+        message = f"Package {file} {deploy_response.description[6]} to Snowflake @{stage}/{file}."
+
+        if deploy_response.description[6] == "SKIPPED":
+            message = "Package already exists on stage. Consider using --overwrite to overwrite the file."
+
+        return message
+
+    def create(self, name: str):
+        file_name = name + ".zip"
+        if os.path.exists(".packages"):
+            utils.recursive_zip_packages_dir(pack_dir=".packages", dest_zip=file_name)
+
+        if os.path.exists(file_name):
+            return CreatedSuccessfully(Path(file_name))
+        else:
+            return CreationError()
+
+    @staticmethod
+    def cleanup_after_install():
+        if os.path.exists(".packages"):
+            rmtree(".packages")
+
+    @staticmethod
+    def create_lookup_message(lookup_result: LookupResult, name: str):
+
+        if type(lookup_result) == InAnaconda:
+            return f"Package {name} is available on the Snowflake anaconda channel."
+        elif type(lookup_result) == RequiresPackages:
+            return f"""The package {name} is supported, but does depend on the
+                    following Snowflake supported native libraries. You should
+                    include the following in your packages: {lookup_result.requirements.snowflake}"""
+        elif type(lookup_result) == NotInAnaconda:
+            return f"""The package {name} is avaiable through PIP. You can create a zip using:\n
+                    snow snowpark package create {name} -y"""
+        else:
+            return f"Lookup for package {name} resulted in some error. Please check the package name and try again"
```

Comment:
How about adding those messages to correspondind classes:
```
class InAnaconda(LookupResult):
     def __init__(self, package_name: str):
         self.package_name = package_name
   

     @property
     def message(self):
         return f"""The package {self.package_name} is supported, but does depend on the
                    following Snowflake supported native libraries. You should
                    include the following in your packages..."
```
WDYT?

Reply:
Done